### DISCUSSION TITLE
WPML - License Key

### DISCUSSION THREAD

**Discussion by davideprevosto** [tombroucke/privatewpackagist#27] [Q&A] [answered]
I'm trying to setup a WPML package. I own a very old Lifetime license, but I can't find anywhere the License Key. I have a User Id of course, a Site Key each site a managed with WPML but I am not aware about a license key.
Anyone could help?

Thanks

**Comment by tombroucke** [comment, marked as answer]
Hi,

The license key is called "Subscription key" in WPML. If you log in to wpml.org, go to downloads, and copy a download link, you can find the subscription key:

https://wpml.org/?download=10420399&user_id={{ user_id }}&subscription_key={{ license_key }}&t=1730103633&version=3.1.3

**Reply by davideprevosto** [reply]
@tombroucke thank you! I will try later this evening.
